src/coord/radar/Radar.js

```diff
@@ -40,7 +41,8 @@ function Radar(radarModel, ecModel, api) {
 
     this._indicatorAxes = zrUtil.map(radarModel.getIndicatorModels(), function (indicatorModel, idx) {
         var dim = 'indicator_' + idx;
-        var indicatorAxis = new IndicatorAxis(dim, new IntervalScale());
+        var indicatorAxis = new IndicatorAxis(dim,
+            (indicatorModel.axisType === 'log') ? new LogScale() : new IntervalScale());
```

Comment:
Use `indicatorModel.get('type')` to access the `type` option in the `indicator` like this:
```js
indicator: [{ type: 'log' }]
```

Reply:
I switched to get as requested.